Machine Learning/Analytics Jupyter Notebooks/ml-analytics-classification.ipynb

```diff
@@ -0,0 +1,469 @@
+      "                                        {'phase': 'loading_data',\n",
+      "                                         'progress_percent': 100},\n",
+      "                                        {'phase': 'analyzing',\n",
+      "                                         'progress_percent': 100},\n",
+      "                                        {'phase': 'writing_results',\n",
+      "                                         'progress_percent': 100}],\n",
+      "                           'state': 'stopped'}]}\n"
+     ]
+    }
+   ],
+   "source": [
+    "# 3. Check the job stats\n",
+    "\n",
+    "stats_endpoint = \"/_ml/data_frame/analytics/model-flight-delay-classification/_stats\"\n",
+    "result = requests.get(es_url+stats_endpoint)\n",
+    "pprint.pprint(result.json())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## View Classification Results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'hits': {'hits': [{'_index': 'df-flight-delayed',\n",
+       "    '_id': '-5L81W4BszKtAdTQ-PZn',\n",
+       "    '_score': 1.0,\n",
+       "    '_source': {'FlightNum': 'K29GWY6',\n",
+       "     'Origin': 'Syracuse Hancock International Airport',\n",
+       "     'OriginLocation': {'lon': '-76.10630035', 'lat': '43.11119843'},\n",
+       "     'DestLocation': {'lon': '128.445007', 'lat': '51.169997'},\n",
+       "     'FlightDelay': True,\n",
+       "     'DistanceMiles': 5774.020251542041,\n",
+       "     'FlightTimeMin': 799.492323179845,\n",
+       "     'OriginWeather': 'Cloudy',\n",
+       "     'dayOfWeek': 6,\n",
+       "     'AvgTicketPrice': 669.8655726244879,\n",
+       "     'Carrier': 'ES-Air',\n",
+       "     'FlightDelayMin': 180,\n",
+       "     'OriginRegion': 'US-NY',\n",
+       "     'FlightDelayType': 'Late Aircraft Delay',\n",
+       "     'DestAirportID': 'XHBU',\n",
+       "     'timestamp': '2019-12-01T02:53:59',\n",
+       "     'Dest': 'Ukrainka Air Base',\n",
+       "     'FlightTimeHour': 13.324872052997417,\n",
+       "     'Cancelled': False,\n",
+       "     'DistanceKilometers': 9292.384847697675,\n",
+       "     'OriginCityName': 'Syracuse',\n",
+       "     'DestWeather': 'Rain',\n",
+       "     'OriginCountry': 'US',\n",
+       "     'ml__id_copy': '-5L81W4BszKtAdTQ-PZn',\n",
+       "     'DestCountry': 'RU',\n",
+       "     'DestRegion': 'RU-AMU',\n",
+       "     'OriginAirportID': 'SYR',\n",
+       "     'DestCityName': 'Belogorsk',\n",
+       "     'ml': {'top_classes': [{'class_probability': 0.919598560379138,\n",
+       "        'class_name': 'true'},\n",
+       "       {'class_probability': 0.08040143962086195, 'class_name': 'false'}],\n",
+       "      'FlightDelay_prediction': 'true',\n",
+       "      'prediction_probability': 0.919598560379138,\n",
+       "      'is_training': True}}}]}}"
```

Comment:
might be a good idea to specifically choose a document where `"is_training": False` to demonstrate that we're not just predicting delays for the flights we've trained on :D

Reply:
Yes :) Good point - I've hopefully fixed that.